src/main/java/uk/gov/ons/census/fwmtadapter/messaging/ActionFieldReceiver.java

```diff
@@ -11,7 +11,6 @@
 import uk.gov.ons.census.fwmtadapter.model.dto.field.ActionAddress;
 import uk.gov.ons.census.fwmtadapter.model.dto.field.ActionInstruction;
 import uk.gov.ons.census.fwmtadapter.model.dto.field.ActionRequest;
-import uk.gov.ons.census.fwmtadapter.model.dto.field.Priority;
```

Comment:
should delete the uk.gov.ons.census.fwmtadapter.model.dto.field.Priority class now that it's no longer used

Reply:
Deleted, had to remove one use of it in the `ActionUpdate`.